### PR TITLE
[JENKINS-38114] Consolidated help for RunWrapper

### PR DESCRIPTION
**Pull request by jglick**
[JENKINS-38114](https://issues.jenkins-ci.org/browse/JENKINS-38114)

Infrastructure only; actual fixes are downstream.

@reviewbybees

### CODE REVIEW

File: src/main/resources/org/jenkinsci/plugins/workflow/support/steps/build/RunWrapper/help.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    <dt><code>buildVariables</code></dt><dd>for a non-Pipeline downstream build, offers access to a map of defined build variables; for a Pipeline downstream build, any variables set globally on <code>env</code></dd>
+    <dt><code>changeSets</code></dt><dd>a list of <a href="http://javadoc.jenkins-ci.org/hudson/scm/ChangeLogSet.html" target="_blank">changesets</a> coming from distinct SCM checkouts; each has a <code>kind</code> and is a list of commits; each commit has a <code>commitId</code>, <code>timestamp</code>, <code>msg</code>, <code>author</code>, and <code>affectedFiles</code> each of which has an <code>editType</code> and <code>path</code>; the value will not generally be <code>Serializable</code> so you may only access it inside a method marked <code>@NonCPS</code></dd>
+    <dt><code>rawBuild</code></dt><dd>a <code>hudson.model.Run</code> with <a href="http://javadoc.jenkins-ci.org/hudson/model/Run.html" target="_blank">further APIs</a>, only for trusted libraries or administrator-approved scripts outside the sandbox; the value will not be <code>Serializable</code> so you may only access it inside a method marked <code>@NonCPS</code></dd>
```
Comment (jglick) [inline src/main/resources/org/jenkinsci/plugins/workflow/support/steps/build/RunWrapper/help.html]: The last two lines edited a bit to emphasize [`workflow-cps` 2.14](https://wiki.jenkins-ci.org/display/JENKINS/Pipeline+Groovy+Plugin#PipelineGroovyPlugin-2.14%28Sep07%2C2016%29).

**Comment by abayer** [comment]
:bee:

**Comment by jglick** [comment]
Also putting in a hotfix for [JENKINS-38457](https://issues.jenkins-ci.org/browse/JENKINS-38457) here, since otherwise the downstream PR is broken.

**Comment by abayer** [comment]
re-:bee:

**Pull request merged**